src/Library/demos/Drop Zone/main.js

```diff
@@ -0,0 +1,124 @@
+import Gtk from "gi://Gtk";
+import Gio from "gi://Gio";
+import Adw from "gi://Adw";
+import Gdk from "gi://Gdk";
+import GObject from "gi://GObject";
+
+const bin = workbench.builder.get_object("bin");
+
+// Universal drop target for any String data
+const string_drop_target = Gtk.DropTarget.new(
+  GObject.TYPE_STRING,
+  Gdk.DragAction.COPY,
+);
+
+bin.add_controller(string_drop_target);
+
+string_drop_target.connect("drop", (self, value, x, y) => {
+  bin.child = createTextWidget(value);
+  bin.get_style_context().remove_class("overlay-drag-area");
```

Comment:
`get_style_context` is deprecated.

Use `add_css_class` and `remove_css_class` Widget methods directly